chain/network/src/network_protocol/proto_conv.rs

```diff
@@ -483,6 +487,13 @@ impl From<&PeerMessage> for proto::PeerMessage {
                     })
                 }
             }),
+            tx_timestamp: MF::from_option(tx_timestamp.ok().map(|duration| {
```

Comment:
please use https://github.com/near/nearcore/blob/2eec03616a924112da587a4da676c4883d1acd27/chain/network-primitives/src/time.rs#L66 
and convert it like this: https://github.com/near/nearcore/pull/7023/files#diff-2e879b433bc841c07dc50b3c7947ebf60988834054e0dd39b74e173b12b3c99d
(unfortunately this PR is not merged yet).